### DISCUSSION TITLE
제페토 앱 상의 월드 방문자 수치 관련 문의

### DISCUSSION THREAD

**Discussion by Joyounwoo** [naverz/zepeto-studio-kor#892] [5. Q&A]
제페토 앱 상에서 월드 방문자 수치 관련 질문이 있어 문의 드립니다.

월드의 리텐션 비율을 확인할 방법이 있을까요? 없다면 기존 제공되는 수치를 가공해서 알아낼 방법이 있을까요?

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재는 월드 콘솔 내 통계 페이지로만 보실 수 있으며 
Google Analytics를 사용할 수 있게 준비 중입니다.
지원 시기는 조율 중으로 확답이 어려운 점 양해 부탁 드립니다. 
감사합니다.
